apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/InpatientClaimTransformerV2Test.java

```diff
@@ -41,75 +67,1592 @@ public InpatientClaim generateClaim() throws FHIRException {
+            2,
+            "3333444555",
+            "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBClaimCareTeamRole",
+            "operating",
+            "Operating");
+
+    Assert.assertTrue(compare2.equalsDeep(member2));
+
+    // Third member
+    CareTeamComponent member3 = TransformerTestUtilsV2.findCareTeamBySequence(3, eob.getCareTeam());
+    CareTeamComponent compare3 =
+        TransformerTestUtilsV2.createNpiCareTeamMember(
+            3,
+            "161943433",
+            "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBClaimCareTeamRole",
+            "otheroperating",
+            "Other Operating");
 
-    assertMatches(
-        claim,
-        InpatientClaimTransformerV2.transform(new MetricRegistry(), claim, Optional.of(false)));
+    Assert.assertTrue(compare3.equalsDeep(member3));
+
+    // Fourth member
+    CareTeamComponent member4 = TransformerTestUtilsV2.findCareTeamBySequence(4, eob.getCareTeam());
+    CareTeamComponent compare4 =
+        TransformerTestUtilsV2.createNpiCareTeamMember(
+            4,
+            "345345345",
+            "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBClaimCareTeamRole",
+            "performing",
+            "Performing provider");
+
+    Assert.assertTrue(compare4.equalsDeep(member4));
   }
 
-  private static final FhirContext fhirContext = FhirContext.forR4();
+  /** SupportingInfo items */
+  @Test
+  public void shouldHaveSupportingInfoList() {
+    Assert.assertEquals(11, eob.getSupportingInfo().size());
+  }
+
+  @Test
+  public void shouldHaveNchPtntStusIndCdSupInfo() {
+    SupportingInformationComponent sic =
+        TransformerTestUtilsV2.findSupportingInfoByCode(
+            "https://bluebutton.cms.gov/resources/variables/nch_ptnt_stus_ind_cd",
+            eob.getSupportingInfo());
+
+    SupportingInformationComponent compare =
+        TransformerTestUtilsV2.createSupportingInfo(
+            // We don't care what the sequence number is here
+            sic.getSequence(),
+            // Category
+            Arrays.asList(
+                new Coding(
+                    "http://terminology.hl7.org/CodeSystem/claiminformationcategory",
+                    "info",
+                    "Information"),
+                new Coding(
+                    "https://bluebutton.cms.gov/resources/codesystem/information",
+                    "https://bluebutton.cms.gov/resources/variables/nch_ptnt_stus_ind_cd",
+                    "NCH Patient Status Indicator Code")),
+            // Code
+            new Coding(
+                "https://bluebutton.cms.gov/resources/variables/nch_ptnt_stus_ind_cd",
+                "A",
+                "Discharged"));
+
+    Assert.assertTrue(compare.equalsDeep(sic));
+  }
+
+  @Test
+  public void shouldHaveAdmissionPeriodSupInfo() {
+    SupportingInformationComponent sic =
+        TransformerTestUtilsV2.findSupportingInfoByCode("admissionperiod", eob.getSupportingInfo());
+
```

Comment:
Andrew. just curious as to whether we should do an assert prior to firing a deep comparison; to wit:

Assert.assertNotNull(sic);

Now granted if it is null the deep compare would fail; I just don't know what the failure state/message might be (i.e., it may be obvious that the compare on the null _sic_.